### DISCUSSION TITLE
Primefaces 14: new Style of focused TextArea

### DISCUSSION THREAD

**Discussion by klippelU** [primefaces/community#2500] [PrimeFaces] [answered]
After update to PF14 the textarea tag in table looks stange after focus. This was done by adding new CSS at `components.css`.

`.ui-datatable-data td[role="gridcell"]:focus,.ui-datatable-data td[role="gridcell"] *:focus,.ui-datatable-data td[role="grid"] [tabindex="0"]:focus{outline:var(--primary-color,#90caf9);outline-style:solid;outline-width:2px;border:0}`

This do override `ui-state-focus` styleClass.

Old Style:
![Textarea old](https://github.com/user-attachments/assets/7b77a891-5d16-4558-8b16-2523bc28eafe)

New Style:
![Textarea new](https://github.com/user-attachments/assets/e95ffd1c-0731-46ba-abf6-7211a95534ac)

Is there any reason for the new style?

**Comment by melloware** [comment]
Yes keyboard navigation: https://github.com/primefaces/primefaces/issues/11783

![datatable-navigation](https://github.com/primefaces/primefaces/assets/4399574/50fb2ce4-c065-4f97-bbfc-5d090fcf4eaa)

**Reply by melloware** [reply, marked as answer]
This is fixed in 14.0.6: https://github.com/primefaces/primefaces/pull/12599
